src/components/TextArea/TextArea.tsx

```diff
@@ -31,19 +35,34 @@ export function TextArea(props: TextAreaProps) {
     <FormControl {...getFormControlProps()}>
       <textarea className={className} {...inputProps} {...rest} />
 
-      {props.maxLength && (
-        <div className={classes.counter}>
-          {currentLength}/{props.maxLength} caracteres
+      <div className={classes.footer}>
+        <div>
+          {error && (
+            <FormError id={errorId} role='alert'>
+              {error}
+            </FormError>
+          )}
+        </div>
```

Comment:
Isn't showing a empty `div` if doesn't have `error`? 

Maybe in this way fix this:

```suggestion
	{error && (
		<div>
			<FormError id={errorId} role='alert'>
				{error}
			</FormError>
		</div>
	)}
```